components/x-gift-article/src/HighlightSection.scss

```diff
@@ -0,0 +1,38 @@
+@import '@financial-times/o-typography/main';
+@import '@financial-times/o-colors/main';
+@import '@financial-times/o-spacing/main';
+
+.x-gift-article__highlight-section {
+	border-top: 1px solid oColorsMix(black, paper, 20);
+	padding-top: oSpacingByName('s6');
+	padding-bottom: oSpacingByName('s6');
+
+	@include oGridRespondTo($from: S) {
+		padding-bottom: oSpacingByName('s3');
+	}
+
+	@include oLabels(
+		$opts: (
+			'states': (
+				'content-premium'
+			)
+		)
+	);
+}
+
+.x-gift-article__form .o-forms-field.x-gift-article__checkbox {
+	margin: 0;
+}
+
+.x-gift-article__checkbox-span::before {
+	background-color: inherit !important;
```

Comment:
**Change request:** Please avoid using `!important`

Reply:
Unfortunately this is the only way i found to enforce this style , any other way gets override by the form class the is imported in next article project for some reason , will discuss with the team if anyone have a solution for this